modules/helper/EsxiServiceHelper.py

```diff
@@ -0,0 +1,84 @@
+from modules.configuration.Configuration import Configuration
+from interfaces.host import Host
+from modules.api.Atlas import Atlas
+from modules.api.SshHelper import SshHelper
+
+from threading import Thread
+from queue import Queue
+
+import logging
+
+
+logger = logging.getLogger('esxi')
+
+
+class EsxiServiceHelper:
+
+    def __init__(self, config: Configuration) -> None:
+        self.atlas = Atlas(config)
+        self.config = config
+
+        self._services = self.config.monitored_serivces
+        command_list = ["/etc/init.d/%s status" %
+                        service for service in self._services]
+        self._command = ' & '.join(command_list)
+
+        # Strange bug. Display name is different from registered service name
+        self._services = [item.replace('nsx-opsagent', 'opsAgent')
+                          for item in self._services]
+
+    def _worker(q: Queue, ssh_username: str, ssh_password: str,
+                command: str, services: list, output: list):
+        while not q.empty():
+            host: Host = q.get()
+
+            answer = SshHelper.execute_command(
+                host.address,
+                ssh_username,
+                ssh_password,
+                command
+            )
+
+            host.services = {}
+            if answer is None:
+                host.services = {k: False for k in services}
+                # Maybe add some info to host that something is wrong?
```

Comment:
You can use `StateSetMetricFamily` here. The metric value is a dict of string state names to booleans.

https://github.com/prometheus/client_python/blob/5dffc77b9ff1884ffcae5142f64d6428c6c05982/prometheus_client/metrics_core.py#L297